### PR TITLE
Include raiden-network/raiden-services in the docker-compose bundle

### PR DESCRIPTION
**Pull request by konradkonrad**
This is a remote fix for https://github.com/raiden-network/raiden-services/issues/551 (first bullet point)

- Add the raiden-services `.env` values to `.env.template`
- Add the raiden-services containers to `docker-compose.yaml`
- ~~Upgrade `docker-compose.yaml` `version` to `3.4` and document necessary changes when running~~
- Update `README.md`

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -30,3 +30,24 @@ WORKER_COUNT=8
...
+
+### Services settings
+## the keystore file, has to be located in ${DATA_DIR}/keystore
+KEYSTORE_FILE=<KEYSTORE_FILE>
+PASSWORD=<PASSWORD>
+ETH_RPC=<ETH_RPC_ENDPOINT>
+
+## The name of the chain
+## For example on the goerli testnet this would become: CHAIN_ID=goerli
+CHAIN_ID=<CHAIN_ID>
```
Comment (ulope) [inline .env.template]: Even more nitpicky: There probably should be no placeholder value

File: README.md [hunk 2]
```diff
@@ -52,14 +52,14 @@ provide configurations with services split among multiple servers.
           |
 +---------v---------+
 |                   |
-|      Traefik      |  
+|      Traefik    +-+-----> Federation to other  
+|                 | |       Raiden Matrix servers
++---------+-------+-+
+          |       |
++---------v-------v-+
```
Comment (ulope) [inline README.md]: This 'graph' should also be updated

File: .env.template [hunk 3]
```diff
@@ -30,3 +30,24 @@ WORKER_COUNT=8
...
+
+
+### Services settings
+## the keystore file, has to be located in ${DATA_DIR}/keystore
+KEYSTORE_FILE=<KEYSTORE_FILE>
+PASSWORD=<PASSWORD>
+ETH_RPC=<ETH_RPC_ENDPOINT>
+
+## The name of the chain
+## For example on the goerli testnet this would become: CHAIN_ID=goerli
```
[the remark this answers is not in the document]
Comment (konradkonrad) [reply .env.template]: @palango can you help me out here? What network names are supported by the services?

[the remark this answers is not in the document]
Comment (palango) [reply .env.template]: See https://github.com/raiden-network/raiden-services/blob/master/src/request_collector/cli.py#L21

However, we're moving to calling this chain_id, so I'll create a PR to rename `NETWORK_ID` to `CHAIN_ID`.

**Comment by konradkonrad** [comment]
> Looks pretty good already.
> 
> I left comments of varying nitpickyness below :)

Addressed many of your nitpicks. Biggest changes are:
- backport to docker-compose `v2.3`
- use of `pfs.` subdomain for pathfinding service
- defined a couple of dummy healthchecks that should be considered `FIXME`s
@ulope could you look through it again?

File: docker-compose.yml [hunk 4]
```diff
@@ -1,27 +1,97 @@
-# Only compose file versions >=2.2,<3 support the `scale` parameter
-version: "2.3"
+# Since we will use '3.4'
```
[the remark this answers is not in the document]
Comment (konradkonrad) [reply docker-compose.yml]: backported to 2.3

File: docker-compose.yml [hunk 5]
```diff
@@ -1,27 +1,97 @@
...
+      - PFS_STATE_DB=/state/pfs-state.db
+      - PFS_HOST=0.0.0.0
+      - SERVER_NAME
+      - PFS_LOG_LEVEL=${LOG_LEVEL}
+      - PFS_KEYSTORE_FILE=/keystore/${KEYSTORE_FILE}
+      - PFS_PASSWORD=${PASSWORD}
+      - PFS_ETH_RPC=${ETH_RPC}
+    labels:
+      - "traefik.enable=true"
+      - "traefik.frontend.rule=Host:${SERVER_NAME}"
```
[the remark this answers is not in the document]
Comment (konradkonrad) [reply docker-compose.yml]: added a subdomain for `pfs.<domain>`

**Pull request merged**